package.json

```diff
@@ -50,10 +48,7 @@
     "extraFiles": [
       "config/Symphony.config",
       "library/libsymphonysearch.dylib",
-      "library/indexvalidator.exec",
-      "library/lz4.exec",
-      "library/search-launch-agent.sh",
-      "library/search-launch-daemon.sh"
+      "library/lz4.exec"
```

Comment:
is this still required? if so, what is it for?

Reply:
Yes, When we build the electron we need to copy these libraries.